src/main/kotlin/com/cognifide/gradle/aem/api/AemConfig.kt

```diff
@@ -340,9 +340,10 @@ class AemConfig(
 
     /**
      * Number of intervals / additional instance stability checks to assure all stable instances.
+     * This mechanism protect against temporary stable states.
      */
     @Input
-    var awaitAssurances: Long = propParser.long("aem.await.assurances", 3L)
+    var awaitAssurances: Long = propParser.long("aem.await.assurances", 5L)
```

Comment:
`awaitDelay` becomes redundant because `awaitAssurances` covers that functionality